gammapy/background/tests/test_energy_offset_array.py

```diff
@@ -0,0 +1,24 @@
+from ..energy_offset_array import EnergyOffsetArray
+from gammapy.data import DataStore
+from gammapy.data import EventList
+from ...datasets import gammapy_extra
+from ...utils.testing import requires_dependency, requires_data
+import numpy as np
+
+
+@requires_dependency('matplotlib')
+@requires_data('gammapy-extra')
+def test_energy_offset_array():
+    dir = str(gammapy_extra.dir) + '/datasets/hess-crab4'
+    data_store = DataStore.from_dir(dir)
+    Observation_Table = data_store.obs_table
```

Comment:
In Python, there are [naming conventions](https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions):
* Modules, functions and variables are `snake_case`.
* Classes are `CamelCase`

I don't know why the `pep8` command line tool doesn't point those out by default, but e.g. [pylint](http://www.pylint.org/) does.
```
$ pylint gammapy/background/tests/test_energy_offset_array.py -r n
No config file found, using default configuration
************* Module gammapy.background.tests.test_energy_offset_array
C:  1, 0: Missing module docstring (missing-docstring)
C: 11, 0: Missing function docstring (missing-docstring)
W: 12, 4: Redefining built-in 'dir' (redefined-builtin)
C: 14, 4: Invalid variable name "Observation_Table" (invalid-name)
C: 22, 8: Invalid variable name "Array" (invalid-name)
W: 18, 9: Unused variable 'i' (unused-variable)
C:  6, 0: external import "import numpy as np" comes before "from ..energy_offset_array import EnergyOffsetArray" (wrong-import-order)
```

In this file you should use `obs_table` instead of `Observation_Table`, and `array` instead of `Array`.

PyCharm will also point this out to you if you open up the file, and it has a refactor -> rename option that even works across multiple files ... using that would be my suggestion, remembering to have to run `pep8` or `pylint` from the command line is too annoying.

Apologies for being super-nitpicky on this pull request ... this is close to ready!